### ISSUE TITLE
Lack of tooltip inside for Kanban

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1810]
This button doesn't have a tooltip and it might be confusing for some users, not knowing what it does.

![Image](https://github.com/user-attachments/assets/1a1e013f-fe43-4fc7-a8f9-2ec6abf5065c)
